t/create-deposit-data.golden

```diff
@@ -28,7 +28,7 @@ Key #3: cNYaH3onqrdMffpznhMMmrHn34fuTU59w5j8LM3H42VPcUsLeXy5
 Key #4: cRoydfinDRzzRQJp5niqJWukSYTfPJQM6ytqGN6nzonaz1mafgwD
 
 Cold storage address:
-2NAVkgszAvjZZwWpHpbCaxSaNqCCXPD8Bts
+2N93du8YobdgsHyu3qgBvSyhGUT52utMNeA
```

Comment:
I'm wondering if we should add a backwards-compatible test for depositing into previously generated deposit legacy addresses?

Reply:
There are tests for withdrawing from legacy addresses. Depositing into any Glacier address is done without the use of GlacierScript, so I'm not sure what else there is to test?

Reply:
fair point, I misunderstood the change here.  This makes sense.